client/src/components/map/style.css

```diff
@@ -157,6 +154,10 @@
     height: calc(100vh - 56px - 100px);
   }
 
+  .mapProfilePreviewInfo {
+    margin-left: 0px;
```

Comment:
is it necessary to have a margin-left if you're applying 0px?

Reply:
Yeah, it overrides the margin-left above (line 58).

Reply:
ah, gotcha